test/apiv2/python/rest_api/test_v2_0_0_image.py

```diff
@@ -89,14 +89,9 @@ def test_pull(self):
 
     def test_create(self):
         r = requests.post(
-            self.podman_url + "/v1.40/images/create?fromImage=alpine&platform=linux/amd64/v8",
-            timeout=15,
-        )
+            self.podman_url + "/v1.40/images/create?fromImage=alpine&platform=linux/amd64/v8", timeout=15)
```

Comment:
made similar comment in another PR, is it worthwhile to move the "v1.40" to a variable?

Reply:
Looking at it again, we could do it but then we would eventually have to edit both this file and api_testcase.py. I guess we could create a local variable in this file?

Reply:
I would prefer the version number be expicate for these compat tests.